kedro-org/kedro-viz
# [KED-2985] Migrate Kedro UI utils

## Description

We're migrating the components we use from [Kedro-UI](https://github.com/quantumblacklabs/kedro-ui/), and this is one of those migrations. Here's the [relevant ticket](https://jira.quantumblack.com/browse/KED-2985) (QB-only access).

## Development notes

I've created a `src/utils/search-utils.js` file, which contains the utils we were using from Kedro-UI (they were all related to searching, hence the file name) and a relevant test file. 

## QA notes

To test, search in the search box on the flow chart or experiment-tracking page. You shouldn't notice any difference. Also, run the relevant tests.

## Checklist
